components/maker/styled.js

```diff
@@ -1,6 +1,12 @@
-import Styled from 'styled-components'
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { Flex, Box, Type, Button, Field } from 'blockstack-ui'
 
-export const SidebarButton = Styled.div`
+//
+// These styles are to be considered tempoary, and to be removed
+// upon release of the updated style guide
+
```

Comment:
Not going to review these as they will go away with the release of @blockstack/ui